packages/client/hmi-client/src/types/operator.ts

```diff
@@ -17,12 +17,16 @@ export enum WorkflowOperationTypes {
 	FUNMAN = 'Funman'
 }
 
-export enum WorkflowStatus {
+export enum OperatorStatusType {
+	DEFAULT = 'default',
+	FOCUS = 'focus', // Hover/drag
+	IN_PROGRESS = 'in progress',
+	SUCCESS = 'success',
 	INVALID = 'invalid',
+	WARNING = 'warning', // Probably won't be used - would there be potential crossover with INVALID?
```

Comment:
I would leave `WARNING` alone for now, but I would remove `FOCUS`.

FOCUS is not a status of the operation process, but is a status of the user's interaction and should be tracked else where.  For example I can be "in progress" and "focus" at the same time because I am dragging my calibrate node ... which doesn't make sense to me.

Reply:
Yeah I asked @YohannParis about this last week. He mentioned that there might be a need for it once we add keyboard shortcuts to navigate through nodes. Not sure if I remember exactly

Reply:
This one is a tough one. In my opinion, `FOCUS` state is a temporary state that the node is and should be reverted to its previous state once out of it. But per @mwdchang point of view, this should be done separately...

I think both solution are valid. I will let @mwdchang and you decide which is best.